packages/fxa-auth-server/lib/senders/emails/renderer.ts

```diff
@@ -10,8 +10,8 @@ import path = require('path');
 const config = require('../../../config').getProperties();
 
 const mjmlConfig: Record<any, any> = {
-  validationLevel: 'strict',
-  // filePath: path.join(__dirname, 'css', 'global.css'),
+  validationLevel: 'soft',
```

Comment:
Wasn't able to fix the CI test failures so turning this validation back to soft for now and we can circle back on the issue in a follow up PR

Reply:
That's fine by me, the `cadReminderFirst` ticket is that sort of catch-all for anything remaining/follow ups (plus after both briefly looking at it, we're not sure what the issue is 🤨), let's just be sure to note on that ticket that we need to address this. Jody gave this the r+ and it LGTM, so I'm going to merge it! 🎉